papers/opencadd-klifs/paper.md

```diff
@@ -0,0 +1,62 @@
+tags:
+  - Python
+  - KLIFS
+  - kinase
+authors:
+  - name: Dominique Sydow^[corresponding author]
+    orcid: 0000-0003-4205-8705
+    affiliation: 1
+  - name: Jaime Rodríguez-Guerra
+    affiliation: 2
+  - name: Andrea Volkamer
+    affiliation: 1
+affiliations:
+ - name: _In Silico_ Toxicology and Structural Bioinformatics, Institute of Physiology, Charité – Universitätsmedizin Berlin, corporate member of Freie Universität Berlin and Humboldt-Universität zu Berlin, Augustenburger Platz 1, 13353 Berlin, Germany
+   index: 1
+ - name: Institution Name
+   index: 2
+date: 27 October 2021
+bibliography: paper.bib
+---
+
+# Summary
+
+Protein kinases are involved in most aspects of cell life due to their role in signal transduction. Dysregulated kinases can cause severe diseases such as cancer, inflammatory and neurodegenerative diseases, which has made them a frequent target in drug discovery for the last decades [@Cohen:2021].
+The immense research on kinases has led to an increasing amount of kinase resources [@Kooistra:2017].
+Among them is the KLIFS database, which focuses on storing and analyzing structural data on kinases and interacting drugs and other small molecules [@Kanev:2021].
+The OpenCADD-KLIFS Python module offers a convenient integration of the KLIFS data into workflows to facilitate computational kinase research.
+
+# Statement of need
+
+[OpenCADD-KLIFS](https://opencadd.readthedocs.io/en/latest/databases_klifs.html) (``opencadd.databases.klifs``) is a part of the [OpenCADD](https://opencadd.readthedocs.io/) package, a collection of Python modules for structural cheminformatics.
+This module offers access to KLIFS data [@Kanev:2021] such as information about kinases, structures, ligands, 
+interaction fingerprints, and bioactivities. 
+KLIFS thereby focuses especially on the ATP binding site, defined as a set of 85 residues and aligned across all structures using a multiple sequence alignment (MSA) [@vanLinden:2014].
+With OpenCADD-KLIFS, KLIFS data can be queried either locally from a KLIFS download or remotely from the KLIFS webserver. 
+The presented module provides identical APIs for the remote and local queries for KLIFS data and streamlines all output into 
+standardized Pandas DataFrames [@pandas] to allow for easy and quick downstream data analyses (\autoref{fig:opencadd_klifs_toc}). This Pandas-focused setup is ideal to work with in Jupyter notebooks [@Kluyver:2016]. 
+
+
+![OpenCADD-KLIFS fetches KLIFS data [@Kanev:2021] offline from a KLIFS download or online from the KLIFS database and formats the output as user-friendly Pandas DataFrames [@pandas].\label{fig:opencadd_klifs_toc}](opencadd_klifs_toc.png)
+
+The KLIFS database offers a REST API including a Swagger API specification [@klifsswagger]. Our module OpenCADD-KLIFS uses bravado [@bravado] to dynamically generate a Python client based on these Swagger API definitions and adds wrappers to enable the following functionalities:
```

Comment:
```suggestion
The KLIFS database offers a REST API compliant with the OpenAPI specification [@klifsswagger]. Our module OpenCADD-KLIFS uses bravado [@bravado] to dynamically generate a Python client based on the OpenAPI definitions and adds wrappers to enable the following functionalities:
```
Swagger initially developed the spec and then it was "donated" to the LSF. More info: https://swagger.io/specification/